report/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Why this instead of the master `.gitignore` ignoring `report/`?

Reply:
This is here so that we can have an empty `report/` for the tests to write to.

Reply:
Ah makes sense then